<?xml version="1.0"?>
<launch>

  <arg name="markerId" default="931"/>
  <arg name="markerSize" default="0.275"/>
  <arg name="marker_frame" default="marker1"/>
  <arg name="ref_frame" default="marker1"/>

  <node pkg="aruco_ros" type="single" name="aruco_single" output="screen">
   <remap from="/camera_info" to="/blackfly/camera_info"/>
   <remap from="/image" to="/blackfly/image_raw"/>
   <param name="image_is_rectified" value="true"/>
   <param name="marker_size" value="$(arg markerSize)"/>
   <param name="marker_id" value="$(arg markerId)"/>
   <param name="reference_frame" value="$(arg ref_frame)"/>
   <param name="camera_frame" value="blackfly_mount_link"/>
   <param name="marker_frame" value="$(arg marker_frame)"/>
  </node>

</launch>
